builtin-functions/_functions.txt

```diff
@@ -1373,3 +1373,5 @@ final class FFI {
 
   private function __construct();
 }
+
+function numa_get_bound_node(): int;
```

Comment:
Since there are no usage examples in the PR, it's hard to understand how this function is used in the user code (PHP).
Maybe a short comment could make things easier.

Reply:
added short comment